poppler-sys/src/vendored_bindings.rs

```diff
@@ -0,0 +1,21 @@
+#![allow(
+    non_upper_case_globals,
+    non_camel_case_types,
+    non_snake_case,
+    improper_ctypes,
+    clippy::useless_transmute,
+    clippy::missing_safety_doc,
+    clippy::should_implement_trait,
+    clippy::transmute_ptr_to_ptr
+)]
+
+pub mod bindings_poppler;
+pub mod bindings_poppler_action;
+pub mod bindings_poppler_annot;
+pub mod bindings_poppler_attachment;
+pub mod bindings_poppler_document;
+pub mod bindings_poppler_form_field;
+pub mod bindings_poppler_layer;
+pub mod bindings_poppler_media;
+pub mod bindings_poppler_movie;
+pub mod bindings_poppler_page;
```

Comment:
instead of doing a `include_str!`, just having modules work so much better for ide linters..
although bindgen folks don't recommend changing stuff outside `OUT_DIR` (which is inside `target/`). But since nothing happens without an explicit flag.. perhaps it's not all that bad - again, given that ide works much much better in this configuration